microservice/security/src/test/java/com/cumulocity/microservice/security/filter/util/HttpRequestUtilsTest.java

```diff
@@ -0,0 +1,34 @@
+package com.cumulocity.microservice.security.filter.util;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class HttpRequestUtilsTest {
+
+    //Base 64 credentials coded: "management/rtanaka@hks-power.co.jp:(Def_Leppard)1"
+    private final static String asciiCharactersCredentials = "Basic bWFuYWdlbWVudC9ydGFuYWthQGhrcy1wb3dlci5jby5qcDooRGVmX0xlcHBhcmQpMQ==";
```

Comment:
according to convention, it should be:
```suggestion
    private static final String ASCII_CHARACTERS_CREDENTIALS = "Basic bWFuYWdlbWVudC9ydGFuYWthQGhrcy1wb3dlci5jby5qcDooRGVmX0xlcHBhcmQpMQ==";
```